web/entities/check.go

```diff
@@ -0,0 +1,46 @@
+package entities
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/trento-project/trento/web/models"
+	"gorm.io/datatypes"
+)
+
+type Check struct {
+	ID        string `gorm:"primaryKey"`
+	CreatedAt time.Time
+	Payload   datatypes.JSON
+}
+
+type CheckList []*Check
+
+// TableName overrides the table name used by CheckRaw to `checks`
+func (Check) TableName() string {
```

Comment:
This shouldn't be strictly necessary as `Check` entity will be inflected by gorm to actually `checks`.
Yet, I don't see a problem in having it explicit as well.

Reply:
You are right. Leftover from previous struct name `CheckRaw`